src/good_fences_runner.rs

```diff
@@ -91,14 +92,28 @@ impl GoodFencesRunner {
         println!("Evaluating {} files", self.source_files.keys().len());
         let mut evaluation_results = FenceEvaluationResult::new();
 
+        let resolver: CachingResolver<TsConfigResolver<NodeModulesResolver>> = CachingResolver::new(
+            60_000,
+            TsConfigResolver::new(
+                NodeModulesResolver::new(TargetEnv::Node, Default::default(), false),
+                ".".into(),
+                self.tsconfig_paths_json
```

Comment:
Maybe instead of using the whole tsconfig.paths.json since the only properties we use are `compilerOptions.baseUrl` and `compilerOptions.paths` maybe passing those two properties as config options instead of the whole file?

Reply:
Will work on this in a separate PR